packages/node/test/utils.test.js

```diff
@@ -0,0 +1,257 @@
+'use strict'
+
+import { test } from 'node:test'
+import { strict as assert } from 'node:assert'
+import { resolve, join } from 'node:path'
+import { mkdtemp, writeFile, rm, mkdir, readFile } from 'node:fs/promises'
+import { tmpdir } from 'node:os'
+import { ignoreDirs, isServiceBuildable } from '../lib/utils.js'
+
+const tempDir = resolve(tmpdir(), 'packages-node-utils')
+
+test('utils - isServiceBuildable', async (t) => {
```

Comment:
Generically it's better to use `describe` for the top level, drop `async` in this line and just use `test` from `node:test`.

no need to fix this now